src/Pages/Authentication/SignUp.js

```diff
@@ -1,7 +1,84 @@
+import { useState } from 'react'
+import { toast, ToastContainer } from 'react-toastify'
+import { useAuth } from '../../Context/AuthContext'
+ import 'react-toastify/dist/ReactToastify.css';
+ import './Auth.css'
+import { useNavigate } from 'react-router-dom';
+
 export const SignUp = () =>{
+    const [firstName, setFirstName] = useState('')
+    const [lastName, setLastName] = useState('')
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+    const { handleSingupData} = useAuth()
+    const [error, setError] = useState('')
+    const navigate = useNavigate()
+    const handleSubmit = () =>{
+        if(firstName && lastName && email && password){
+            if(password.length >=8){
+                handleSingupData(firstName, lastName, email, password)
+            }
+            else{
+                setError('password should be greater than 8 characters')
+            }
+        }
+        else{
+            setError('Please fill all the fields')
+            }
+    }
+    
     return(
-        <div>
-            SignUp page
+        <div className="auth-container">
+            <div class="form">
+                <span className='bold-text'>Sign up</span>
+                <div class="input-with-icons ">
+                    <i class="bi bi-person-fill input-icon"></i>
+                    <input type="email" 
+                    placeholder="Enter your First Name" 
+                    class="icon-input"
+                    value={firstName}
+                    onChange={(e)=>setFirstName(e.target.value)}/>
+                </div>
```

Comment:
Suggestions: here you are creating signup form, try to wrap inside a "form" html tag. Use semantic html.